draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1679,6 +1679,9 @@ AEGIS-128X and AEGIS-256X share the same security properties and requirements as
 
 AEGIS is considered secure against guess-and-determine attacks aimed at recovering the state from observed ciphertexts. This resilience extends to quantum adversaries in the Q1 model, wherein quantum attacks do not confer any practical advantage for decrypting previously recorded ciphertexts or achieving key recovery.
 
+This document extends the original specification by introducing optional support for 256-bit authentication tags, which are constructed similarly to the 128-bit tags.
+As shown in {{SSI24}}, with 256-bit tags, all AEGIS variants achieve more than 128-bit forgery security by differential attacks.
```

Comment:
Is this meant to be on a new line? Please ignore this question if the same answer applies to the other PR.

Also `forgery security by differential attacks` sounds a bit awkward to me. Could we rephrase this? Is `forgery security` a common term?

Reply:
Ok my bad. I've signed in to view the paper and it says that so let's leave it. I've just normally heard 'security against forgery'.

Reply:
We can replace `forgery security` by `security against forgery`. It feels indeed more natural.